packages/cli/src/lib/Compiler.ts

```diff
@@ -393,7 +417,10 @@ export class Compiler {
       project.getManifestDir(),
       imageName,
       dockerfile,
-      project.quiet
+      cacheDir,
+      buildxOutput,
+      project.quiet,
```

Comment:
I think having `quiet: boolean` be the last argument of the function makes sense for both `createBuildImage` and `copyArtifactsFromBuildImage`